app/forms/questionnaires/senco_start_date.rb

```diff
@@ -0,0 +1,41 @@
+module Questionnaires
+  class SencoStartDate < Base
+    include ActiveRecord::AttributeAssignment
+    include Helpers::Institution
+
+    QUESTION_NAME = :senco_start_date
+
+    attr_reader QUESTION_NAME
+
+    def senco_start_date=(value)
```

Comment:
Would it be possible to use here `ActiveModel` `attribute` method (macro) with type `:date` instead of manually casting?